src/main/resources/db/migration/V013__Make_message_id_nullable.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE letters
+ALTER COLUMN message_id DROP NOT NULL;
```

Comment:
dropping should be separate pr as reading (application) is published after the migration is done

Reply:
this is dropping a constraint, column becomes nullable, it's fine.

Reply:
ignore me :|